src/router.js

```diff
@@ -46,6 +47,17 @@ var Router = {
       }
     });
 
+    try {
+      let rulesEngine = require('rules-engine/index.js');
+      // Issue a token without an associated user
+      JSONWebToken.issueToken(-1).then(jwt => {
```

Comment:
So this JWT isn't associated with a user? How does that work exactly? What happens when the user logs out? What about when another user logs in? Just asking because I don't understand the JWT stuff very well.